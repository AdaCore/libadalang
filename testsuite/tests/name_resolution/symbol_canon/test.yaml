driver: name-resolution
charset: utf-8
input_sources: [foo.adb]
