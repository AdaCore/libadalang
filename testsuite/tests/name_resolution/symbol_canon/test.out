Analyzing foo.adb
#################

\u03a0 resolves to:
    \u03c0        : constant Float := 3.14;
\xc9 resolves to:
    ["00E9"] : Wide_Character := '["00E9"]';

Done.
